src/craterstats/constants.py

```diff
@@ -5,15 +5,7 @@
 import matplotlib.path as mpath
 import craterstats.gm as gm
 
-try:
-    import importlib.resources as importlib_resources
-except ImportError:
-    # Try backported to PY<37 `importlib_resources`.
-    import importlib_resources
-
-
-with importlib_resources.path("craterstats.config", "demo_commands.txt") as path:
-    PATH = gm.filename(str(path),'u')
+PATH='src/craterstats/'
```

Comment:
hey @ggmichael, so I think the changes in this PR prevents zip resources from working. specifically the lines above this one that use importlib _should_ work as is to be zip safe